app/views/i18n/I18NEditCourseView.js

```
@@ -13,85 +1,140 @@
 
 // TODO: Apply these changes to all i18n views if it proves to be more reliable
 
-module.exports = (I18NEditCourseView = (function () {
-  I18NEditCourseView = class I18NEditCourseView extends I18NEditModelView {
-    static initClass () {
-      this.prototype.id = 'i18n-edit-course-view'
-      this.prototype.modelClass = Course
-    }
+const I18NEditCourseView = class I18NEditCourseView extends I18NEditModelView {
+  static initClass () {
+    this.prototype.id = 'i18n-edit-course-view'
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)